Interface/IImageService.cs

```diff
@@ -1,7 +1,11 @@
-﻿namespace mPole.Interface
+﻿using Microsoft.AspNetCore.Components.Forms;
+using mPole.Data.Models;
+
+namespace mPole.Interface
 {
     public interface IImageService
     {
         string ImageBase64(byte[] imageData);
+        Task<List<Image>> UploadImagesAsync(List<IBrowserFile> files, string moveName);
```

Comment:
Lepiej w takich sytuacjach używać Task<IEnumerable> lub Task<IList> - niby pierdoła - sam czasami używam List - ale no dobra praktyka przy Liskov i SP to interfejsy